### ISSUE TITLE
测试时前几个测试点TLE[BUG]

### ISSUE THREAD

**Issue by wind-whisperfxj** [Project-LemonLime/Project_LemonLime#227]
**Describe the bug**
**描述一下这个 Bug**
评测过程中每道题评测时总会有前几个测试点超时，且每次重测超时的测试点数量不固定，而不超时的时候运行时间远小于时限。  

**To Reproduce**
**如何复现**
进行任意题目的评测，不仅是新建的题目，也包括之前表现正常的cdf文件中的题目。

**Expected behavior**
**期望获得的结果**
不要出现异常超时

**Screenshots**
**有截图吗？**
以下为两次重测的结果
![微信截图_20240129020414](https://github.com/Project-LemonLime/Project_LemonLime/assets/93211525/f37dae63-4c40-4c18-856f-1787065fcad9)

![微信截图_20240129020430](https://github.com/Project-LemonLime/Project_LemonLime/assets/93211525/8f8686af-10fa-4c23-8b46-0f1da93e1e51)

**Environment:**
**环境：**
-   Version: v0.3.4.4 & v0.2.3
-   OS: 系统：
Windows 11 家庭中文版

**Additional context**
**其他信息**
通过添加重复测试点看出与测试点内容无关，只与测试点位置有关。
看到了[#224](https://github.com/Project-LemonLime/Project_LemonLime/issues/224)中的类似问题，但本机并没有安装火绒等杀毒软件。
Lemon并非新安装，半年前还能正常使用，期间和Lemon相关的变化是将其从C盘桌面迁移到了D盘根目录，但目前从D盘删除，重装到原位置，并不能解决问题。
尝试回退版本，重装v0.2.3，未能解决问题。
尝试将编译器从默认的 D:\mingw64\mingw64\bin\g++.exe 修改成DevC++附带的 C:\Program Files (x86)\Dev-Cpp\MinGW64\bin\g++.exe ，未能解决问题。

**Comment by d0j1a1701** [comment]
把你杀软关了

![image](https://github.com/Project-LemonLime/Project_LemonLime/assets/48874489/c1a44edc-903e-453d-9814-a94014efd0ac)

![image](https://github.com/Project-LemonLime/Project_LemonLime/assets/48874489/992957b8-8c24-408d-ab77-edb81d365baf)

**Comment by wtcsky** [comment]
好像要关闭杀毒软件才行

**Comment by Black-HIG** [comment]
Windows Defender 对于一些应用（例如刚被 gcc 编译出的 exe）会进行云扫描，耗时很长
![image](https://github.com/user-attachments/assets/6b76b926-38c1-441a-b78c-329ce3ca7b07)
所以前几个点会 TLE 掉

**Comment by mycbxzd1** [comment]
确实,因此我们需要关掉Windows Defender甚至包括火绒在内的一些杀毒软件,我认为作者应该注意更改这些内容,但是在原lenmon中并未出现此类问题,因此应更新代码 

Indeed, we have to disable Windows Defender and even some antivirus software like Huorong. I believe the author should pay attention to modifying these aspects. However, such issues did not occur in the original Lemon, so the code should be updated accordingly.

**Comment by mycbxzd1** [comment]
经过询问ChatGPT, 我们给出如下解决方案（尤其是在比赛中）：  
After consulting ChatGPT, we propose the following solutions (especially during competitions):  

1. **排除文件夹**：将用于编译和运行代码的文件夹添加到 Windows Defender 的排除列表中。步骤如下：  
1. **Exclude Folder**: Add the folder used for compiling and running code to the exclusion list of Windows Defender. Steps are as follows:  

- 打开“Windows 安全中心”。  
- Open "Windows Security Center."  

- 选择“病毒和威胁防护”。  
- Select "Virus & Threat Protection."  

- 点击“管理设置”下的“排除项”。  
- Click "Exclusions" under "Manage Settings."  

- 添加编译器输出目录或整个编译目录为排除项，这样 Windows Defender 将不再扫描该文件夹中的文件。  
- Add the compiler output directory or the entire compilation directory to the exclusions so that Windows Defender will no longer scan files in that folder.  

2. **排除特定进程**：将 `g++.exe` 或者与评测相关的进程排除在 Windows Defender 的扫描范围外。  
2. **Exclude Specific Process**: Exclude `g++.exe` or other processes related to evaluation from the Windows Defender scanning range.  

- 打开“Windows 安全中心”。  
- Open "Windows Security Center."  

- 选择“病毒和威胁防护”。  
- Select "Virus & Threat Protection."  

- 找到“排除项”，并添加 `g++.exe` 进程。  
- Find "Exclusions" and add the `g++.exe` process.  

3. **关闭实时保护**（临时）：在评测过程中可以暂时关闭 Windows Defender 的实时保护，这样可以避免实时扫描影响评测时间。  
3. **Turn Off Real-time Protection** (Temporarily): During the evaluation, you can temporarily disable Windows Defender's real-time protection to avoid real-time scanning from affecting evaluation time.  

- 打开“Windows 安全中心”。  
- Open "Windows Security Center."  

- 选择“病毒和威胁防护”。  
- Select "Virus & Threat Protection."  

- 关闭“实时保护”。（注意：关闭实时保护会有安全风险，建议只在评测期间短暂关闭。）  
- Turn off "Real-time Protection." (Note: Disabling real-time protection poses security risks, so it's recommended to do so only briefly during the evaluation period.)

**Comment by Black-HIG** [comment]
> 确实,因此我们需要关掉Windows Defender甚至包括火绒在内的一些杀毒软件,我认为作者应该注意更改这些内容,但是在原lenmon中并未出现此类问题,因此应更新代码
> 
> Indeed, we have to disable Windows Defender and even some antivirus software like Huorong. I believe the author should pay attention to modifying these aspects. However, such issues did not occur in the original Lemon, so the code should be updated accordingly.

我觉得我们应该去找编译器/WD的问题而不是Lemon的问题

Update：
经过测试后，旧版 Lemon 仍然触发 bug
![image](https://github.com/user-attachments/assets/e7a9e891-5731-46e8-a6d6-3c412d73c3a6)
